composer.json

```diff
@@ -6,42 +6,46 @@
     "license": "MIT",
     "require": {
         "php": "^7.3",
-        "sylius/sylius": "^1.7",
-        "symfony/messenger": "^4.3"
+        "sylius/sylius": "^1.9"
     },
     "require-dev": {
         "behat/behat": "^3.6.1",
-        "behat/mink": "^1.7@dev",
-        "behat/mink-browserkit-driver": "^1.3",
-        "behat/mink-extension": "^2.2",
-        "behat/mink-selenium2-driver": "^1.3",
+        "behat/mink-selenium2-driver": "^1.4",
+        "dmore/behat-chrome-extension": "^1.3",
+        "dmore/chrome-mink-driver": "^2.7",
+        "friends-of-behat/mink": "^1.8",
+        "friends-of-behat/mink-browserkit-driver": "^1.4",
+        "friends-of-behat/mink-extension": "^2.4",
         "friends-of-behat/page-object-extension": "^0.3",
         "friends-of-behat/suite-settings-extension": "^1.0",
-        "friends-of-behat/symfony-extension": "^2.0",
-        "friends-of-behat/variadic-extension": "^1.1",
-        "lakion/mink-debug-extension": "^1.2.3",
-        "phpspec/phpspec": "^6.0",
-        "phpstan/phpstan-doctrine": "^0.12",
-        "phpstan/phpstan": "^0.12",
-        "phpstan/phpstan-webmozart-assert": "^0.12",
-        "phpunit/phpunit": "^7.0",
+        "friends-of-behat/symfony-extension": "^2.1",
+        "friends-of-behat/variadic-extension": "^1.3",
+        "friendsofsymfony/oauth-server-bundle": "^1.6 || >2.0.0-alpha.0 ^2.0@dev",
+        "lakion/mink-debug-extension": "^2.0.0",
+        "phpspec/phpspec": "^7.0",
+        "phpstan/extension-installer": "^1.0",
+        "phpstan/phpstan": "0.12.82",
+        "phpstan/phpstan-doctrine": "0.12.33",
+        "phpstan/phpstan-strict-rules": "^0.12.0",
+        "phpstan/phpstan-webmozart-assert": "0.12.12",
+        "phpunit/phpunit": "^9.5",
         "sensiolabs/security-checker": "^6.0",
-        "sylius-labs/coding-standard": "^3.0",
-        "symfony/browser-kit": "^4.4",
-        "symfony/debug-bundle": "^4.4",
-        "symfony/dotenv": "^4.4",
-        "symfony/intl": "^4.4",
-        "symfony/web-profiler-bundle": "^4.4",
-        "symfony/web-server-bundle": "^4.4"
+        "sylius-labs/coding-standard": "^3.1",
```

Comment:
`vendor/bin/ecs check src` doesn't work with the current dependency.  Do you mean  `"sylius-labs/coding-standard": "^4.0"`?

Reply:
3.1 version is from plugin-skeleton. when I try on 4.0 version,  gh actions don't build correctly